src/workers/Picker.java

```diff
@@ -1,17 +1,39 @@
 package workers;
 
+import java.util.*;
+import
 import vehicles.Forklift;
+import stocking.*;
+
+/*
+ * The Picker class, a worker in the warehouse
+ */
 
 public class Picker extends Worker {
 
 	private Forklift forklift;
+	
+	/*
+	 * Initialize a Picker object
+	 */
 
-	public Picker(String name) {
-		super(name);
+	public Picker(String name, Warehouse warehouse) {
+		super(name, warehouse);
 		this.forklift = new Forklift();
 	}
+	
+	/*
+	 * Complete the designated picker task of picking stock
+	 */
 
-	public void doTask() {
-		;
+	public void doTask(String location) {
```

Comment:
This has changed from what it used to be, now the location argument is going to only be a single location, and should be of the form "ZONE_LETTER AISLE_INT RACK_INT LEVEL_INT"